### PR TITLE
Fixes #14657: Check output of each directories system checker output

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: test/fixtures/checks/pass/pass.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+exit 0
```
Comment (domcleal) [inline test/fixtures/checks/pass/pass.sh]: Missing a shebang line, this seems to fail under Ruby 1.8.7.

File: test/fixtures/checks/fail/fail.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+exit 1
```
Comment (domcleal) [inline test/fixtures/checks/fail/fail.sh]: Should have a shebang line as well, to be safe.

**Comment by domcleal** [comment]
:+1: to the fix, it works correctly.  #143 will fix the other tests on 1.8.7.

**Comment by ehelms** [comment]
Shebangs added

**Comment by domcleal** [comment]
Thanks, the new tests are passing now.  Can be merged once master's passing.

**Comment by ares** [comment]
[test]

**Comment by ares** [comment]
Thanks @ehelms, merging!

**Pull request merged**